code-of-conduct-languages/bn.md

```diff
@@ -1,74 +1,84 @@
+* কারও সম্মতি ছাড়াই অনুসরণ করা বা অনুসরণ করা
+* অবাঞ্ছিত শারীরিক যোগাযোগ
+* অবাঞ্ছিত যৌন বা রোমান্টিক মনোযোগ বা অগ্রগতি
+* অন্যান্য আচরণ যা পেশাদার পরিবেশে যুক্তিসঙ্গতভাবে অনুপযুক্ত বলে
+  বিবেচিত হতে পারে
 
-* যৌনতাপূর্ণ ভাষা বা চিত্রের ব্যবহার, এবং যৌন মনোযোগ বা
-  যেকোনো ধরনের অগ্রগতি
-* ট্রোলিং, অপমানজনক বা অবমাননাকর মন্তব্য এবং ব্যক্তিগত বা রাজনৈতিক আক্রমণ
-* সরকারি বা ব্যক্তিগত হয়রানি
-* অন্যের ব্যক্তিগত তথ্য প্রকাশ করা, যেমন একটি শারীরিক বা ইমেল
-  ঠিকানা, তাদের সুস্পষ্ট অনুমতি ছাড়া
-* অন্যান্য আচরণ যা যুক্তিসঙ্গতভাবে অনুপযুক্ত হিসাবে বিবেচিত হতে পারে একটি পেশাদার সেটিং 
+নিম্নলিখিত আচরণগুলিও নিষিদ্ধ:
 
+* আচরণবিধির তদন্তের সাথে সম্পর্কিত জেনেশুনে মিথ্যা বা বিভ্রান্তিকর তথ্য সরবরাহ করা বা অন্যথায় ইচ্ছাকৃতভাবে তদন্তে হস্তক্ষেপ করা।
+* কোনও ব্যক্তির বিরুদ্ধে প্রতিশোধ নেওয়া কারণ তারা কোনও ঘটনা রিপোর্ট করেছে বা সাক্ষী হিসাবে কোনও ঘটনা সম্পর্কে তথ্য সরবরাহ করেছে।
 
+প্রকল্প রক্ষণাবেক্ষণকারীদের মন্তব্য, প্রতিশ্রুতি, কোড, উইকি সম্পাদনা, সমস্যা এবং অন্যান্য অবদান যা এই আচরণবিধির সাথে সামঞ্জস্যপূর্ণ নয় তা অপসারণ, সম্পাদনা বা প্রত্যাখ্যান করার অধিকার এবং দায়িত্ব রয়েছে।
+এই আচরণবিধি গ্রহণ করে, প্রকল্প রক্ষণাবেক্ষণকারীরা একটি সিএনসিএফ প্রকল্প পরিচালনার প্রতিটি ক্ষেত্রে এই নীতিগুলি ন্যায্য এবং ধারাবাহিকভাবে প্রয়োগ
+করার জন্য প্রতিশ্রুতিবদ্ধ।
+প্রকল্প রক্ষণাবেক্ষণকারীরা যারা আচরণবিধি অনুসরণ বা প্রয়োগ করে না তাদের প্রকল্প দল থেকে অস্থায়ীভাবে বা স্থায়ীভাবে অপসারণ করা যেতে পারে।
 
-এই কোড অফ কন্ডাক্টের সাথে সংযুক্ত নয় এমন মন্তব্য, প্রতিশ্রুতি, কোড, উইকি সম্পাদনা (wiki edits), সমস্যা (issue) এবং অন্যান্য অবদানগুলি অপসারণ, সম্পাদনা বা প্রত্যাখ্যান করার অধিকার এবং দায়িত্ব প্রকল্প রক্ষণাবেক্ষণকারীদের  রয়েছে।
+## প্রতিবেদন
 
-এই আচরণবিধি অবলম্বন করে, প্রকল্পের রক্ষণাবেক্ষণকারীরা প্রতিটি ক্ষেত্রে এই নীতিগুলিকে ন্যায্যভাবে এবং ধারাবাহিকভাবে প্রয়োগ করতে প্রতিশ্রুতিবদ্ধ  এই প্রকল্প পরিচালনার প্রজেক্ট রক্ষণাবেক্ষণকারী যারা কোড অব কন্ডাক্ট অনুসরণ বা প্রয়োগ করেন না
-তাদের স্থায়ীভাবে প্রকল্প দল থেকে সরানো হতে পারে । 
+কুবারনেটস সম্প্রদায়ের মধ্যে ঘটে যাওয়া ঘটনাগুলির জন্য, <conduct@kubernetes.io> মাধ্যমে [কুবারনেটস কোড অফ কন্ডাক্ট কমিটি] (https://git.k8s.io/community/committee-code-of-conduct) এর সাথে যোগাযোগ করুন। আপনি তিন কার্যদিবসের মধ্যে একটি প্রতিক্রিয়া আশা করতে পারেন।
 
+অন্যান্য প্রকল্পের জন্য, বা প্রকল্প-অজ্ঞেয়বাদী বা একাধিক সিএনসিএফ প্রকল্পকে প্রভাবিত করে এমন ঘটনাগুলির জন্য, দয়া করে <conduct@cncf.io> এর মাধ্যমে [সিএনসিএফ কোড অফ কন্ডাক্ট কমিটি] (https://www.cncf.io/conduct/committee/) এর সাথে যোগাযোগ করুন।  বিকল্পভাবে, আপনি আপনার প্রতিবেদন জমা দেওয়ার জন্য [সিএনসিএফ কোড অফ কন্ডাক্ট কমিটি] (https://www.cncf.io/conduct/committee/) এর যে কোনও স্বতন্ত্র সদস্যের সাথে যোগাযোগ করতে পারেন। বেনামে কীভাবে রিপোর্ট জমা দিতে হয়, সেই সহ কীভাবে রিপোর্ট জমা দিতে হয়, সেই বিষয়ে আরও বিস্তারিত নির্দেশাবলীর জন্য অনুগ্রহ করে আমাদের [ইনসিডেন্ট রেজোলিউশন প্রসিডিউরস](https://github.com/cncf/foundation/blob/main/code-of-conduct/coc-incident-resolution-procedures.md) দেখুন। আপনি তিন কার্যদিবসের মধ্যে একটি প্রতিক্রিয়া আশা করতে পারেন।
```

Comment:
অন্যান্য প্রকল্পের জন্য, বা প্রকল্প-অজ্ঞেয়বাদী বা একাধিক CNCF প্রকল্পকে প্রভাবিত করে এমন ঘটনাগুলির জন্য, দয়া করে <conduct@cncf.io> এর মাধ্যমে [CNCF কোড অফ কন্ডাক্ট কমিটি](https://www.cncf.io/conduct/committee/🖐 এর সাথে যোগাযোগ করুন। বিকল্পভাবে, আপনি আপনার প্রতিবেদন জমা দেওয়ার জন্য [CNCF কোড অফ কন্ডাক্ট কমিটি](https://www.cncf.io/conduct/committee/🖐 এর যে কোনও স্বতন্ত্র সদস্যের সাথে যোগাযোগ করতে পারেন।  বেনামে কীভাবে রিপোর্ট জমা দিতে হয়, সেই সহ কীভাবে রিপোর্ট জমা দিতে হয়, সেই বিষয়ে আরও বিস্তারিত নির্দেশাবলীর জন্য অনুগ্রহ করে আমাদের [ইনসিডেন্ট রেজোলিউশন প্রসিডিউরস](https://github.com/cncf/foundation/blob/main/code-of-conduct/coc-incident-resolution-procedures.md🖐 দেখুন। আপনি তিন কার্যদিবসের মধ্যে একটি প্রতিক্রিয়া আশা করতে পারেন।



replace 🖐 with )